raiden/network/blockchain_service.py

```diff
@@ -29,7 +30,13 @@
 
 
 class BlockChainService:
-    """ Exposes the blockchain's state through JSON-RPC. """
+    """ Exposes the blockchain's state through JSON-RPC.
+
+    This class is used to synchronize the calls to the blockchain. An example for
```

Comment:
This additional comment is incorrect. This class basically caches instances for certain proxies depending on the values used to instantiate the proxies with.

Example: 
- Token Proxy uses the token address as the cache key
- Token Network Registry Proxy uses the token network registry address ... etc

And additional block-related utils apparently.

Reply:
Good point, I updated the docstring.

In my opinion it would also make sense to rename the class to `ProxyManager` to make the purpose clearer (and later get rid of the rpc client related functions)